backend/routes.py

```
@@ -15,9 +15,11 @@
 from routes_chemicals import register_chemical_routes
 from routes_chemical_analytics import register_chemical_analytics_routes
 from routes_calibration import register_calibration_routes
+from routes_bulk_import import register_bulk_import_routes
 from routes_rbac import register_rbac_routes, permission_required
```

Flake8 (7.2.0)

[error] 'routes_rbac.permission_required' imported but unused

(F401)